lil_aretomo/utils.py

```diff
@@ -14,16 +14,18 @@ def prepare_output_directory(
         tilt_angles: List[float],
         basename: str,
         pixel_size: float,
+        skip_if_completed: bool
```

Comment:
```suggestion
```

Reply:
Ah, ok, without this flag it means that prepare_output_directory will automatically skip writing out the mrc files if they exist: although isn't that skipping without asking as the user can't control whether it's skipped or not? Either way, should be a moot point as I doubt people are going the be changing TS names in between running the same job with the same output

Reply:
Have left a comment - we should always write the TS to avoid potential weird hard to debug bugs